src/components/ItemFilter.vue

```diff
@@ -88,7 +124,8 @@ export default {
   },
   data() {
     return {
-      sortOrder: 1,
+      itemSearch: null,
+      sortOrder: 'asc',
```

Comment:
Changing the sortOrder from int to string seems to not be fully working (what was the reason for it?). I'm getting:

> Invalid prop: type check failed for prop "value". Expected String with value "0", got Number with value 0.

Reply:
Yeah this was/is a fiddily one. Basically what I was experimenting with was trying to get things more aligned with the STAC spec out of the box - ATM the filter has to pass things through a formatter to put it into a format compliant with the API's. 
 https://github.com/radiantearth/stac-browser/blob/e780a3a1625ba0684fabfef2d4fcca252a9228a1/src/components/ItemFilter.vue#L195-L202
I think I also ran into issues with one of the API's (MSC PC probably) not respecting the `+` or `-` values, and only accepting `asc` or `desc`.

When I implemented the changes I wasn't thinking about sorting of [catalog](https://github.com/radiantearth/stac-browser/blob/v3/src/components/Catalogs.vue#L48-L53) or [items](https://github.com/radiantearth/stac-browser/blob/v3/src/components/Items.vue#L94-L99) lists, which happen outside of an API.

So I'll have a think about how to reconcile this.

Reply:
Have fallen back to the old approach.